### DISCUSSION TITLE
svg measuraments

### DISCUSSION THREAD

**Discussion by crisfab** [arkypita/LaserGRBL#2004] [General]
How can I modify the measurements of an .svg and the coordinates? In other formats it does allow modification but not in svg, why?

Thanks

**Comment by StuartB4** [comment]
You need to do it in the program it was created with or something like inkscape, which is free.  If you used inkscape to make the image you need to set the size / scale measurements etc in inkscape and then save it, and load it in to LaserGRBL, because LaserGRBL is not an image editing program, it's for streaming the gcode to the engraver.
